db/schema.rb

```diff
@@ -339,6 +339,14 @@
     t.datetime "updated_at"
   end
 
+  create_table "members_messages", id: false, force: :cascade do |t|
+    t.integer "message_id", limit: 4, null: false
+    t.integer "member_id",  limit: 4, null: false
+  end
+
+  add_index "members_messages", ["member_id", "message_id"], name: "index_members_messages_on_member_id_and_message_id", using: :btree
+  add_index "members_messages", ["message_id", "member_id"], name: "index_members_messages_on_message_id_and_member_id", using: :btree
+
```

Comment:
@AndyGauge can you remove this part from the diff? (I think it's part of your other PR.)